### PR TITLE
[ci] Migrate webui::project_helper to rspec

### PR DESCRIPTION
**Pull request by hennevogel**

### CODE REVIEW

File: src/api/spec/helpers/webui/project_helper_spec.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    skip
+  end
+
+  describe '#short_incident_name' do
+    skip
+  end
+
+  describe '#patchinfo_rating_color' do
+    it 'returns the right color' do
+      expect(patchinfo_rating_color('important')).to eq('red')
```
Comment (ChrisBr) [inline src/api/spec/helpers/webui/project_helper_spec.rb]: Why not testing the other cases?
(At least I would test the case when nothing is found in the hash)

File: src/api/spec/helpers/webui/project_helper_spec.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    end
+  end
+
+  describe '#patchinfo_category_color' do
+    it 'returns the right color' do
+      expect(patchinfo_category_color('security')).to eq('maroon')
```
Comment (ChrisBr) [inline src/api/spec/helpers/webui/project_helper_spec.rb]: same here

File: src/api/spec/helpers/webui/project_helper_spec.rb [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    end
+  end
+
+  describe '#incident_issue_color' do
+    skip
+  end
+
+  describe '#map_request_state_to_flag' do
+    it 'returns the right flag' do
+      expect(map_request_state_to_flag('new')).to eq('flag_green')
```
Comment (ChrisBr) [inline src/api/spec/helpers/webui/project_helper_spec.rb]: same here

File: src/api/spec/helpers/webui/project_helper_spec.rb [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    end
+    it 'returns no flag if passed nothing' do
```
Comment (ChrisBr) [inline src/api/spec/helpers/webui/project_helper_spec.rb]: new line missing

File: src/api/spec/helpers/webui/project_helper_spec.rb (hunk 1 above)
Comment (bgeuken) [inline src/api/spec/helpers/webui/project_helper_spec.rb]: +1
We should test all cases here

File: src/api/spec/helpers/webui/project_helper_spec.rb (hunk 3 above)
Comment (hennevogel) [inline src/api/spec/helpers/webui/project_helper_spec.rb]: It's tested below

File: src/api/spec/helpers/webui/project_helper_spec.rb (hunk 1 above)
Comment (hennevogel) [inline src/api/spec/helpers/webui/project_helper_spec.rb]: done

File: src/api/spec/helpers/webui/project_helper_spec.rb (hunk 2 above)
Comment (hennevogel) [inline src/api/spec/helpers/webui/project_helper_spec.rb]: done

**Comment by hennevogel** [comment]
Peepz?

**Comment by mdeniz** [comment]
LGTM :+1:

**Comment by mdeniz** [comment]
Awesome :smile_cat:

**Pull request merged**